Pipfile

```diff
@@ -7,7 +7,7 @@ name = "pypi"
 structlog = "*"
 behave = "*"
 pika = "*"
-census-rm-sample-loader = {editable = true,git = "https://github.com/ONSdigital/census-rm-sample-loader",ref = "v2.0.0"}
+census-rm-sample-loader = {editable = true,git = "https://github.com/ONSdigital/census-rm-sample-loader",ref = "make-log-frequency-configurable"} # TODO replace with release version
```

Comment:
Will be updated to `v2.1.0` once the linked sample loader change is merged and released.

Reply:
👍